tests/test_ide_integration.py

```diff
@@ -0,0 +1,112 @@
+#    Copyright 2018 D-Wave Systems Inc.
```

Comment:
This copyright might need to be in 2020 as your code is new. I will double-check with legal though. (just making a note here in case I forget)